kodilla-exception/src/main/java/com/kodilla/exception/main/ExceptionModuleRunner.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.exception.main;
+
+import com.kodilla.exception.test.SecondChallenge;
+
+public class ExceptionModuleRunner {
+
+    public static void main(String args[]) {
+
+        SecondChallenge secondChallenge = new SecondChallenge();
+
+        try {
+            secondChallenge.probablyIWillThrowException(1,1);
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
W zadaniu byla prosba zeby wyjatek obsluzyc w klasie ExceptionHandling w tym samym pakiecie co SecondChallenge. Detal, ale robi sie przez to balagan